CONTRIBUTING.md

```
@@ -0,0 +1,107 @@
+We use a **Scoped Label System** (`prefix: value`).  
+Each issue should ideally contain:
+  
+- **1× type**
+- **1× priority**
+- **1× area**
+- **1× layer**
+- **0–2× status**
+
+---
+### 🟦 Type: What kind of work is this?
+| Label                                                                      | Description                                          |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)